qbatch/qbatch.py

```diff
@@ -369,6 +369,15 @@ def qbatchDriver(**kwargs):
     system = kwargs.get('system')
     env_mode = kwargs.get('env')
     shell = kwargs.get('shell')
+    block = kwargs.get('block')
+
+    if block:
+        if system == 'sge':
+            options.append('sync -y')
+        elif system == 'pbs':
+            options.append('-Wblock=true')
+        elif system == 'slurm':
+            options.append('--wait')
```

Comment:
These appends need to have a space at the front of them so they don't concat against the other options and break everything.

Reply:
Fixed